tests/introspection/Mac/MacApiSelectorTest.cs

```diff
@@ -138,6 +138,13 @@ protected override bool Skip (Type type, string selectorName)
 					if (!Mac.CheckSystemVersion (10, 12)) // NSCoding was added in 10.12
 						return true;
 					break;
+				case "MLDictionaryFeatureProvider":
+				case "MLMultiArray":
+				case "MLFeatureValue":
+				case "MLSequence":
+					if (Mac.CheckSystemVersion (10, 15)) // Fail on Catalina, pass in older OS
+						return true;
```

Comment:
that's skipping OS <= 10.15
if it pass on older you need a different check, like
```csharp
// Fail on Catalina, pass in older OS	
if (Mac.CheckSystemVersion (10, 14))
	break;
if (Mac.CheckSystemVersion (10, 15)) 
	return true;
// 11.x
break;
```